### ISSUE TITLE
Default Mac full screen key combo does not work for non-apple keyboards

### ISSUE THREAD

**Issue by dvdplm** [warpdotdev/Warp#4967]
### Dupe Check

- [X] I have [searched Warp feature requests](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3A%22Feature+requests%22) and there are no duplicates

### Describe the solution you'd like?

The default keyboard short cut for entering full screen is "globe + F", which is a problem for anyone using a non-apple keyboard. 
Please consider changing the default key combo or at the very least make it configurable in the app.

### How important is this feature to you?

2

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Hi @dvdplm Thanks for submitting. As an alternative, you can also use `CMD-CTRL-F` for fullscreen on Mac, and this should work with non-mac keyboards (usually CMD is replaced by WIN or SUPER key).


To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by dvdplm** [comment]
> you can also use `CMD-CTRL-F` for fullscreen on Mac, and this should work with non-mac keyboards

Fair point! Still, it would be nice if the key combo shown in the menu reflected the presence or not of an apple proprietary keys. :)
